projects/1.20/assets/enderite-cj-edition/enderite/lang/zh_cn.json

```
@@ -0,0 +1,106 @@
+{
+  "creativemodetab.enderite.enderite_tab": "末影合金",
+
+  "enchantment.enderite.night_vision": "夜视",
+  "enchantment.enderite.no_gravity": "无重力",
+  "enchantment.enderite.void_floating": "虚空漂浮",
+  "enchantment.enderite.night_vision.desc": "赋予玩家夜视效果",
+  "enchantment.enderite.no_gravity.desc": "使物品失重",
+  "enchantment.enderite.void_floating.desc": "保护物品不受虚空影响",
+
+  "item.enderite.endercrystal_shard": "末影水晶碎片",
+  "item.enderite.ender_dust": "末影尘",
+  "item.enderite.raw_enderite": "末影合金粗矿",
```

```suggestion
  "item.enderite.raw_enderite": "粗末影合金",
```
请参照原版翻译
**此类问题请全文通查**